docs/guidance/rayservice.md

```diff
@@ -131,15 +135,15 @@ $ kubectl port-forward service/rayservice-sample-head-svc 8265
 ```
 Access the dashboard using a web browser at `localhost:8265`.
 
-### Update Ray Serve Deployment Graph
+### Update Ray Serve config
```

Comment:
Users may not know this is equivalent to "In-place update for Ray Serve applications".